ee-9/feature-pack/src/main/resources/modules/system/layers/base/org/hibernate/jipijapa-hibernate6/main/module.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please move this file to https://github.com/wildfly/wildfly/tree/main/ee-9/common